tensorboard_plugin_wit/wit_plugin.py

```diff
@@ -50,6 +51,8 @@
 # Max number of mutants to show per feature (i.e. num of points along x-axis).
 NUM_MUTANTS = 10
 
+# Max number of examples to send in a single response.
+MAX_EXAMPLES_TO_SEND = 100000
```

Comment:
Since this file imports platform utils, it may be better to define MAX_EXAMPLES_TO_SEND there and then just refer to it here. Right now you define a separate batch_size there and from my understanding they match?

Reply:
added TODO.

the batch_size in platform_utils is just for the call to servo which has different max size semantics, so keeping those separate. added comment about it.